src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -97,6 +105,16 @@ def construct_invocation_parameters(
         validate_invocation_parameters(supported_params, formatted_invocation_parameters)
         return formatted_invocation_parameters
 
+    @classmethod
+    def dependencies_are_available(cls) -> bool:
+        try:
+            for dependency in cls.dependencies():
+                if importlib.util.find_spec(dependency) is None:
+                    return False
+            return True
+        except Exception:
+            return False
```

Comment:
Is the try-except needed?

Reply:
I don't think so but I really don't know this function so I put it there out of fear lol

Reply:
might be able to cache this with `lru_cache` decorator

Reply:
find_spec is very fast so I'm not really worried